dev/NavigationView/NavigationView.xaml

```diff
@@ -294,11 +283,7 @@
                                                     <ScrollViewer VerticalScrollBarVisibility="Auto">
                                                         <local:ItemsRepeater
                                                                     AutomationProperties.AccessibilityView = "Content"
-                                                                    x:Name="TopNavMenuItemsOverflowHost">
-                                                            <local:ItemsRepeater.Layout>
-                                                                <local:StackLayout Orientation="Vertical"/>
```

Comment:
Why remove this?

Reply:
It was @ranjeshj request because vertical stacklayout is default layout

Reply:
@ojhad was there a reason for using a separate instance of stacklayout ? Do we update it at some point. If this is an out of box stack layout, then we can just use the default. But if we are updating it, then we should leave the code as it was since that will end up affecting the shared stack layout object.

Reply:
Ah I see, makes sense. No there is no specific reason for using a separate instance, keeping at default makes sense.